uiowa/uiowa
# Feature: Fix for superfish menu width

This PR addresses: 

https://github.com/uiowa/uiowa/issues/1801

# How to test

Switch https://sitenowdemo.local.drupal.uiowa.edu/ to `uids_base` and add a child page to the about page.   Adjust the theme settings so that the site is using the toggle menu option.   The open menu should look like:

<img width="1298" alt="Screen Shot 2020-06-22 at 4 07 57 PM" src="https://user-images.githubusercontent.com/1036433/85335418-970af200-b4a2-11ea-82e9-2931f804b102.png">
